pom.xml

```diff
@@ -72,6 +73,16 @@
         <artifactId>opentracing-util</artifactId>
         <version>${version.io.opentracing}</version>
       </dependency>
+      <dependency>
+        <groupId>io.opentracing.contrib</groupId>
+        <artifactId>opentracing-ejb</artifactId>
+        <version>0.0.3-SNAPSHOT</version>
```

Comment:
This is because we have a new method on the interceptor. A new PR will be sent once this PR here gets into an advanced stage.